Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManager.cpp

```diff
@@ -113,4 +153,14 @@ namespace AzToolsFramework
         return m_menus[menuIdentifier].GetMenu();
     }
 
+    QMenuBar* MenuManager::GetMenuBar(const AZStd::string& menuBarIdentifier)
+    {
+        if (!m_menuBars.contains(menuBarIdentifier))
+        {
+            return nullptr;
+        }
+
+        return m_menuBars[menuBarIdentifier].GetMenuBar();
```

Comment:
update: Same comment as above, this is one of those performance pessimisation things, using find and the iterator is more efficient (may not matter in this exact case but it's useful to know about for code in a hot path)